templates/jobs/base.html

```diff
@@ -1,5 +1,14 @@
 {% extends 'global/base.html' %}
 {% load staticfiles %}
+{% block stylesheets %}
+    <link rel="stylesheet" href="//brick.a.ssl.fastly.net/Linux+Libertine:400,600,700,400i,600i,700i/Roboto:300,400,300i,400i">
+    <link rel="stylesheet" type="text/css" href="{% static 'vendor/css/bootstrap.min.css' %}" media="screen">
+    <link rel="stylesheet"
+              href="http://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-theme.css">
+    <link rel="stylesheet" type="text/css" href="{% static 'vendor/css/sss.css' %}" media="screen">
+    <link rel="stylesheet" type="text/css" href="{% static 'css/global.css' %}" media="screen">
+    <link rel="stylesheet" type="text/css" href="{% static 'css/community.css' %}" media="screen">
+{% endblock stylesheets %}
 
```

Comment:
Instead of repeating the global stylesheets in jobs/base, you could add a block for extra stylesheets and just include those - so global/base would have an empy ```{% block extra_stylesheets %}{% endblock %}```, and jobs/base would have just the 2 extra stylesheets and would inherit the rest from global/base